src/fleet/azext_fleet/_params.py

```diff
@@ -41,3 +42,28 @@ def load_arguments(self, _):
 
     with self.argument_context('fleet member update') as c:
         c.argument('update_group')
+
+    with self.argument_context('fleet updaterun') as c:
+        c.argument('name', options_list=['--name', '-n'], help='Specify name for the update run.')
+        c.argument('fleet_name', options_list=['--fleet-name', '-f'], help='Specify the fleet name.')
+
+    with self.argument_context('fleet updaterun create') as c:
+        c.argument('upgrade_type', validator=validate_upgrade_type)
+        c.argument('kubernetes_version', validator=validate_kubernetes_version)
+        c.argument('stages', type=file_type, completer=FilesCompleter())
+        c.argument('update_group')
+
+    with self.argument_context('fleet member update') as c:
+        c.argument('update_group')
+
+    with self.argument_context('fleet member update') as c:
+        c.argument('update_group')
+
+    with self.argument_context('fleet updaterun') as c:
+        c.argument('name', options_list=['--name', '-n'], help='Specify name for the update run.')
+        c.argument('fleet_name', options_list=['--fleet-name', '-f'], help='Specify the fleet name.')
+
+    with self.argument_context('fleet updaterun create') as c:
+        c.argument('upgrade_type', validator=validate_upgrade_type)
+        c.argument('kubernetes_version', validator=validate_kubernetes_version)
+        c.argument('stages', type=file_type, completer=FilesCompleter())
```

Comment:
If you use codegen, we can support json, yaml files as well as shorthand sytax for stages property.

Reply:
ok, we have more incoming changes. can I give that a try then?

Reply:
can we do this in other iterations?
especially if your codegen doesn't support typespec generated swagger yet?